src/components/index/index.js

```diff
@@ -92,12 +91,39 @@ console.log('isHome',this.props);
         </nav>
     
         <div className="container fh5co-hero-inner">
-          <h1 className="animated fadeIn wow" data-wow-delay="0.4s">A new tool for purity</h1>
-          <p className="animated fadeIn wow" data-wow-delay="0.67s">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Et voluptates, aliquid soluta distinctio dolorum tenetur. </p>
-          <button className="btn btn-md download-btn-first wow fadeInLeft animated" data-wow-delay="0.85s" onclick="$('#fh5co-download').goTo();return false;"><FontAwesomeIcon icon={faFileDownload}></FontAwesomeIcon>&nbsp;&nbsp;Brochure</button>
-          <button className="btn btn-md features-btn-first animated fadeInLeft wow" data-wow-delay="0.95s" onclick="$('#fh5co-features').goTo();return false;"><FontAwesomeIcon icon={faPuzzlePiece}/>&nbsp;&nbsp;Products</button>
+          <h2 className="product-tagline" data-wow-delay="0.4s">Water filters for taps and showers</h2>
+          <div className="container">
+          <div className="number-section-item row ">
+          <div className="col-md-3 xs-12">
+          <div className="number-section-number-container">
+          <h1 className="grid-number-title-number-section">
+          <CountUp
+  start={0}
+  end={9871}
+  duration={2.75}
+   /> + </h1>
+          <h2 className="grid-sub-title-number-section">Products sold </h2>
+          </div>
+          
+          </div>
+          <div className="col-md-3 xs-12">
+          
+          <div className="number-section-number-container">
```

Comment:
Remove second numbers section, only need one numbers section with an image of smiley